src/Symfony/Bundle/DebugBundle/composer.json

```diff
@@ -23,9 +23,12 @@
     },
     "require-dev": {
         "symfony/config": "~2.8|~3.0",
-        "symfony/dependency-injection": "~2.8|~3.0",
+        "symfony/dependency-injection": "~3.3",
```

Comment:
shouldn't it be a normal requirement ? The bundle is all about defining services anyway.